### ISSUE TITLE
My custom theme is making me sad

### ISSUE THREAD

**Issue by windowsrefund** [tinted-theming/tinty#66]
**What version of `tinty` are you using?**
v0.20.0

Hello again,

I'm having some trouble with my custom theme. Various attempts to understand and troubleshoot have led to #61 but I could use some help just the same. Before getting into things, it's important to note all non-custom themes are working as expected. Here's my custom theme:

```
> tinty list --custom-schemes
base16-prayscale
>
```

And the contents of `prayscale.yaml` itself:

```
# A custom scheme for https://github.com/tinted-theming/tinty
# This is based on grayscale-dark with highlights from
# https://github.com/ramojus/mellifluous.nvim
system: "base16"
name: "Prayscale"
author: "windowsrefund (https://github.com/windowsrefund/)"
variant: "dark"
palette:
  base00: "141414"
  base01: "242424"
  base02: "464646"
  base03: "525252"
  base04: "aeaeae"
  base05: "b9b9b9"
  base06: "e3e3e3"
  base07: "f7f7f7"
  base08: "be7c7d"
  base09: "999999"
  base0A: "f7f7f7"
  base0B: "bfaf8e"
  base0D: "686868"
  base0C: "938ca8"
  base0E: "747474"
  base0F: "5e5e5e"
```

The first odd behavior I'm seeing is the crazy verbosity output to the console when I apply the theme:

```
> tinty apply base16-prayscale
Successfully generated "base24" themes for "base24" at "/home/windowsrefund/.local/share/tinted-theming/tinty/repos/tinted-shell/scripts/*.sh"
Successfully generated "base16" themes for "base16" at "/home/windowsrefund/.local/share/tinted-theming/tinty/repos/tinted-shell/scripts/*.sh"
Successfully generated "base24" themes for "base24-256" at "/home/windowsrefund/.local/share/tinted-theming/tinty/repos/tinted-alacritty/colors-256/*.toml"
Successfully generated "base16" themes for "base16" at "/home/windowsrefund/.local/share/tinted-theming/tinty/repos/tinted-alacritty/colors/*.toml"
Successfully generated "base24" themes for "base24" at "/home/windowsrefund/.local/share/tinted-theming/tinty/repos/tinted-alacritty/colors/*.toml"
Successfully generated "base16" themes for "base16-256" at "/home/windowsrefund/.local/share/tinted-theming/tinty/repos/tinted-alacritty/colors-256/*.toml"
Successfully generated "base16" themes for "default" at "/home/windowsrefund/.local/share/tinted-theming/tinty/repos/base16-i3-colors/themes/*.config"
Successfully generated "base16" themes for "bar-colors" at "/home/windowsrefund/.local/share/tinted-theming/tinty/repos/base16-i3-colors/bar-colors/*.config"
Successfully generated "base16" themes for "colors" at "/home/windowsrefund/.local/share/tinted-theming/tinty/repos/base16-i3-colors/colors/*.config"
Successfully generated "base16" themes for "client-properties" at "/home/windowsrefund/.local/share/tinted-theming/tinty/repos/base16-i3-colors/client-properties/*.config"
Successfully generated "base16" themes for "colors" at "/home/windowsrefund/.local/share/tinted-theming/tinty/repos/base16-i3-client-properties/colors/*.config"
Successfully generated "base16" themes for "default" at "/home/windowsrefund/.local/share/tinted-theming/tinty/repos/base16-i3-client-properties/themes/*.config"
Successfully generated "base16" themes for "bar-colors" at "/home/windowsrefund/.local/share/tinted-theming/tinty/repos/base16-i3-client-properties/bar-colors/*.config"
Successfully generated "base16" themes for "client-properties" at "/home/windowsrefund/.local/share/tinted-theming/tinty/repos/base16-i3-client-properties/client-properties/*.config"
Successfully generated "base16" themes for "base16-powershell" at "/home/windowsrefund/.local/share/tinted-theming/tinty/repos/tinted-fzf/powershell/*.ps1"
Successfully generated "base16" themes for "base16-fish" at "/home/windowsrefund/.local/share/tinted-theming/tinty/repos/tinted-fzf/fish/*.fish"
Successfully generated "base24" themes for "base24-powershell" at "/home/windowsrefund/.local/share/tinted-theming/tinty/repos/tinted-fzf/powershell/*.ps1"
Successfully generated "base24" themes for "base24-default" at "/home/windowsrefund/.local/share/tinted-theming/tinty/repos/tinted-fzf/sh/*.sh"
Successfully generated "base16" themes for "legacy" at "/home/windowsrefund/.local/share/tinted-theming/tinty/repos/tinted-fzf/bash/*.config"
Successfully generated "base16" themes for "base16-default" at "/home/windowsrefund/.local/share/tinted-theming/tinty/repos/tinted-fzf/sh/*.sh"
Successfully generated "base24" themes for "base24-fish" at "/home/windowsrefund/.local/share/tinted-theming/tinty/repos/tinted-fzf/fish/*.fish"
Successfully generated "base16" themes for "base16" at "/home/windowsrefund/.local/share/tinted-theming/tinty/repos/tinted-tmux/colors/*.conf"
Successfully generated "base24" themes for "base24" at "/home/windowsrefund/.local/share/tinted-theming/tinty/repos/tinted-tmux/colors/*.conf"
Successfully generated "base16" themes for "old" at "/home/windowsrefund/.local/share/tinted-theming/tinty/repos/tinted-rofi/themes/*.config"
Successfully generated "base16" themes for "colors" at "/home/windowsrefund/.local/share/tinted-theming/tinty/repos/tinted-rofi/colors/*.rasi"
Successfully generated "base16" themes for "default" at "/home/windowsrefund/.local/share/tinted-theming/tinty/repos/tinted-rofi/themes/*.rasi"
Successfully generated "base16" themes for "default" at "/home/windowsrefund/.local/share/tinted-theming/tinty/repos/base16-kakoune/colors/*.kak"
Successfully generated "base16" themes for "default" at "/home/windowsrefund/.local/share/tinted-theming/tinty/repos/base16-dunst/themes/*.dunstrc"
>
```
I don't get that output when I apply any of the baked-in themes like `base16-grayscale-dark` or `base16-tomorrow-night`. Is this extra verbosity expected when applying a custom theme?

Moving onto the real problem though.... In the following screenshot, I can't explain where the yellow is coming from (it's not in my custom theme)

![tinty-shell](https://github.com/user-attachments/assets/28c7578d-1cf4-4869-9de1-a811704f751b)

Here's the `config.toml` entry related to `tinty-shell` (if it matters)

```
[[items]]
path = "https://github.com/tinted-theming/tinted-shell"
name = "tinted-shell"
themes-dir = "scripts"
hook = ". %f"
```

More than happy to provide additional information as needed. Also worth mentioning perhaps is the fact my shell is setup to leverage [tinty_source_shell_theme](https://github.com/tinted-theming/tinty/blob/main/USAGE.md#sourcing-scripts-that-set-environment-variables) function and corresponding alias as mentioned in the [USAGE](https://github.com/tinted-theming/tinty/blob/main/USAGE.md) documentation.

Basically, I just want to figure out where that yellow !@$%!% is coming from.

I hate yellow

**Comment by JamyGolden** [comment]
I'm busy with a release now (`0.20.1`) which should fix the `Successfully generated "base24"...` isues you're dealing with.

Haha I thin the yellow is coming from the `"base0B": "bfaf8e"` which is a sort of gold colour:
![image](https://github.com/user-attachments/assets/1a4bfff5-c92d-4d8b-b29b-efc10f1e6308)

Is that it or am I missing what you're referring to?

**Comment by windowsrefund** [comment]
Interesting. Will double check tonight. If so, my bad! I appreciate the :eyes:  and will look forward to `0.20.1`. Thanks.

**Comment by windowsrefund** [comment]
You are 100% correct about `base0B` and I'm just noticing how I (falsely) perceive the color to be more yellow than it actually is when used as a foreground compared to when I see it rendered as a background. Interesting. Thanks for the :eyes:

**Comment by JamyGolden** [comment]
No problem, and yeah that is interesting! 😊
